src/Sylius/Bundle/CurrencyBundle/Twig/CurrencyExtension.php

```diff
@@ -41,6 +41,7 @@ public function getFilters()
         return array(
             new \Twig_SimpleFilter('sylius_currency', array($this, 'convertAmount')),
             new \Twig_SimpleFilter('sylius_price', array($this, 'convertAndFormatAmount')),
+            new \Twig_SimpleFilter('sylius_original_price', array($this, 'convertAndFormatAmount')),
```

Comment:
I would remove this and just use 'sylius_price' where you need to. I'm not sure that there's any point having 2 filters doing the same thing and it might even be misleading. I'm trying to think of a use-case for performing a different calculation on originalPrice compared to price and I can't think of one...